src/spec-node/devContainersSpecCLI.ts

```diff
@@ -622,8 +623,8 @@ async function doBuild({
 			}
 		} else if ('dockerComposeFile' in config) {
 
-			if (buildxPlatform || buildxPush) {
-				throw new ContainerError({ description: '--platform or --push not supported.' });
+			if (buildxPush) {
```

Comment:
We don't seem to apply `buildxPlatform` to `docker-compose build` like we do for `docker buildx build`. So it's still unsupported?

Reply:
Now that `buildParams.platformInfo` exists, I expected one of two things to happen:
1. Allow `docker compose` to use the platforms defined in `compose.yml`, and the edits to `inspectImage` etc. will operate on the image with the correct platform, or
2. I'd see a test failure.

But perhaps there's not a test for `docker compose build` with `platforms: []` in the composefile?

Reply:
I don't see a CLI option for Docker Compose to pass the platform along. It might be best for us to keep it unsupported. 🤔